### DISCUSSION TITLE
Measuring the fidelity of SHAP (Kernel SHAP)

### DISCUSSION THREAD

**Discussion by rank-p** [shap/shap#1905] [Q&A]
Hi all,

this may be a trivial issue, but I can't wrap my head around it. I'm performing research in credit scoring and want to measure the fidelity of Kernel SHAP to a given black box model. 
So I have two questions:
1. Is there a common way to measure the fidelity of SHAP, since it is not a model itself but simply attributes the feature impacts?
2. If there is a common measure, is this implemented in the python implementation or do you know about any other open source code which performs this measurement?

Thank you so much in advance!

**Comment by koo-ec** [comment]
I would suggest to check this [repository](https://github.com/amparore/leaf) or this paper: 

Amparore, E., Perotti, A., & Bajardi, P. (2021). To trust or not to trust an explanation: using LEAF to evaluate local linear XAI methods. PeerJ Computer Science, 7, e479.
